### ISSUE TITLE
[Accessibility Audit] Batch Changes: (Admin) Configure webhooks for a code host connection

### ISSUE THREAD

**Issue by umpox** [sourcegraph/sourcegraph#34090]
### Steps to replicate journey

Taken from the [user journey list document](https://docs.google.com/document/d/1XgG0udYL37G3RxuT5shizulpHgb6xVtY5L9kBeSkXYk/edit?usp=sharing). Use this for further context.

- Click your user profile avatar in the top right corner
- Click “Site admin” from the dropdown menu
- Click “Manage code hosts” from the sidebar, within the “Repositories” section
- Click “Edit” for the code host connection
- Click “Add webhook”, or follow the steps to edit the configuration file to enable webhooks for GitHub/BitBucket/GitLab
- Click “Update repositories”
- Copy the webhook URL displayed beneath the button
- Read the steps to set up the webhook from GitHub/BitBucket/GitLab
  - Note: We do not need to audit these steps as we cannot ensure the WCAG status of external sites.

### How to audit

Follow the instructions here: [Auditing a user journey](https://docs.sourcegraph.com/dev/background-information/web/accessibility/how-to-audit#auditing-a-user-journey)

**Note:** We are planning on arranging thorough third-party audit in the future, so our focus here is just to find the _majority_ of accessibility issues. Don't worry if you feel you haven't found 100% of potential issues, it is better to focus on the core essentials to complete the journey rather than spending lots of time going through every possible problem.

### How to raise discovered problems

Follow the instructions here: [Raising an accessibility bug](https://docs.sourcegraph.com/dev/background-information/web/accessibility/how-to-audit#raising-a-bug)

**Comment by LawnGnome** [comment]
Blocked by #36329.

**Comment by LawnGnome** [comment]
General: There's no obvious way to keyboard navigate out of Monaco once you're in it. I've filed #36327 for that.

VoiceOver: I don't think the experience of editing the code host configuration is any worse than it would be to edit any other blob of JSON.

WCAG: The external services page is heavily affected by #32379. The edit external service page also has one instance of #32379, and the Monaco role error at #35715.
